src/Logic/Dictionaries/NamesList.cs

```diff
@@ -159,13 +162,13 @@ public bool Remove(string name)
                     _namesMultiList.Remove(name);
 
                 var userList = new HashSet<string>();
-                var fileName = GetLocalNamesUserFileName();
-                LoadNamesList(fileName, userList, userList);
```

Comment:
Normally yes... but in "Remove" it's intended to load all entries in just one list